app/src/main/java/com/practicum/playlistmaker/player/presentation/PlayerFragment.kt

```diff
@@ -74,11 +66,16 @@ class PlayerFragment : Fragment(R.layout.fragment_player) {
         view.findViewById<ImageView>(R.id.return_n).setOnClickListener {
             requireActivity().onBackPressedDispatcher.onBackPressed()
         }
-    }
-    
-    private fun getTimeString(time: Long): String {
-        return SimpleDateFormat("mm:ss", Locale.getDefault())
-            .format(time)
+
+        viewLifecycleOwner.lifecycleScope.launch {
+            repeatOnLifecycle(Lifecycle.State.STARTED) {
+                viewModel.playerStateFlow.collect {
+                    when (it) {
+                        is PlayerScreenState.TrackTime -> trackTimeTextView.text = it.currentTime
+                    }
+                }
+            }
+        }
```

Comment:
Правильно, но есть запись чуть по-короче:
```suggestion
        viewModel.playerStateFlow
            .flowWithLifecycle(viewLifecycleOwner.lifecycle, Lifecycle.State.STARTED)
            .onEach {
                when (it) {
                    is PlayerScreenState.TrackTime -> trackTimeTextView.text = it.currentTime
                }
            }
            .launchIn(viewLifecycleOwner.lifecycleScope)
```